livekit-agents/livekit/agents/voice_assistant/voice_assistant.py

```diff
@@ -637,12 +637,15 @@ def _commit_user_question_if_needed() -> None:
                 play_handle = answer_synthesis.play()
                 await play_handle.join()
 
-                collected_text = answer_synthesis.collected_text
+                collected_text = answer_synthesis.tr_fwd.played_text
                 interrupted = answer_synthesis.interrupted
 
         if speech_info.add_to_chat_ctx and (not user_question or user_speech_committed):
             self._chat_ctx.messages.extend(extra_tools_messages)
 
+            if interrupted:
+                collected_text += "..."
```

Comment:
This is kind of opinionated, wdyt?

Reply:
maybe the singular ellipsis `…` instead of `...` works better?

Reply:
I think it's fine.. perhaps make this a constant? (if folks want to customize, we could offer it as an option)

Reply:
if folks want to customize it, they can listen to the agent_speech_interrupted event

Reply:
I'd like this to be customized :)